### DISCUSSION TITLE
Can I use Iris standalone with Sodium jar?

### DISCUSSION THREAD

**Discussion by matthaigh27** [IrisShaders/Iris#416] [Q&A] [answered]
Can I use Iris standalone with Sodium jar?
I tried with the latest versions and it didn't work, whats stopping this from working?

**Comment by ghost** [comment, marked as answer]
> I tried with the latest versions and it didn't work, whats stopping this from working?

Did you try regular sodium or sodium w/iris compatibility? (https://github.com/IrisShaders/sodium-fabric)
